tests/e2e/utils.js

```diff
@@ -52,6 +44,23 @@ export function getExtensionElement(id) {
   return $(`>>>[data-qa="${id}"]`);
 }
 
+export async function waitForIdleBackgroundTasks() {
+  const protocol = browser.isChromium ? 'chrome-extension' : 'moz-extension';
+
+  if (!(await browser.getUrl()).startsWith(protocol)) {
+    throw new Error(
+      'Background idle state must be checked from the extension context',
+    );
+  }
+
+  // Wait for the 'idleOptionsObservers' response
+  await browser.execute(
+    browser.isChromium
+      ? () => chrome.runtime.sendMessage({ action: 'idleOptionsObservers' })
```

Comment:
I think in chrome you should be able to evaluate a code in the background. but maybe this is better for now as we will have same mechanism for both platforms (so hopefully will have less surprises)

Reply:
If we would like to call some function in the background it must be available in global context - I think this pattern is clearer, as requires less changes on the background.